jerry-core/jrt/jrt.h

```diff
@@ -108,7 +113,14 @@ extern void __noreturn jerry_unreachable (const char *, const char *, const uint
     } \
   } while (0)
 
+#ifdef __GNUC__
 #define JERRY_UNREACHABLE() __builtin_unreachable ()
+#else /* !__GNUC__ */
+#define JERRY_UNREACHABLE() { \
+#include <stdlib.h> \
+abort(); \
```

Comment:
The question still stands: do we need to do anything here?

    extern void __noreturn jerry_unreachable (const char *, const char *, const uint32_t);

In debug mode, the unreachable function calls a real function, which yields an error and terminates the process. In release mode the undefinied behaviour is expected. Hence why we just do nothing here? Why do we increase the binary size by adding abort calls here?